content/workshops/openshift_101_dcmetromap/lab1-welcome.md

```diff
@@ -73,7 +48,7 @@ You just created a project using the web console, let's tell the terminal comman
 > <i class="fa fa-terminal"></i> Type the following command to use the demo project:
 
 ```bash
-$ oc new-project demo-{{< span userid "YOUR#" >}}
+$ oc project demo-{{< span userid "YOUR#" >}}
```

Comment:
good catch - in workshops I've run students always seem to not be in the right project and having issues. I assumed they had just skipped this step, but it's actually a typo.